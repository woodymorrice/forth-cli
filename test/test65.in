."pi" 3.14 variable pi . . ."pie" ."yum" set . . .
