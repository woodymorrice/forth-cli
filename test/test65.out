3.1400
stack empty
yum
pie
stack empty
